script.js

```diff
@@ -0,0 +1,168 @@
+// DOM 조작(검색)
+const form = document.querySelector('#input-form');
+const input = document.querySelector('#input-form input');
+const yetList = document.querySelector('#yet-list');
+const doneList = document.querySelector('#done-list');
+const todoNum = document.querySelector('#todoNum');
+const doneNum = document.querySelector('#doneNum');
+
+// 텍스트는 변수로 설정 => 오타 방지 및 수정 용이
+const yetText = 'yetList';
+const doneText = 'doneList';
+const yetBtnText = 'yetBtn';
+
+// for localStorage
+let yetArr = [];
+let doneArr = [];
+let id = 0;
+
+// 사용한 함수들에 대한 기능명세는 다음과 같음.
+// 함수1, setValue in localStorage
+const setYetLocal = (arr) => {
+  localStorage.setItem(yetText, JSON.stringify(arr));
+};
+// 함수2, setValue in localStorage
+const setDoneLocal = (arr) => {
+  localStorage.setItem(doneText, JSON.stringify(arr));
+};
+
+// 함수3, render in yetList
+const setYet = (obj) => {
+  // DOM 조작(생성)
+  const li = document.createElement('li');
+  const span = document.createElement('span');
+  const delImg = document.createElement('img');
+  const yetBtn = document.createElement('button');
+  // span.innerText = obj.contents;
+  delImg.src = './img/bin.png';
+  yetBtn.className = yetBtnText;
+  yetBtn.innerText = obj.contents;
+  li.id = obj.id;
+  li.append(yetBtn);
+  li.append(span);
+  li.append(delImg);
+  yetList.append(li); // dom 삽입
+  // del , done 처리
+
+  // 숫자 업데이트
+
+  delImg.addEventListener('click', del);
+  yetBtn.addEventListener('click', done);
+
+  updateNum();
+};
+
+// 함수4, render in doneList
+const setDone = (obj) => {
+  const li = document.createElement('li');
+  const span = document.createElement('span');
+  span.innerText = obj.contents;
+  li.id = obj.id;
+  li.append(span);
+  doneList.append(li);
+  updateNum();
+};
+
+// 함수 5, delete data both render and localStorage
+const del = (e) => {
```

Comment:
```suggestion
const deleteTodoItem = (e) => {
```
함수 이름이 너무 축약된 감이 있죠? 다른 사람이 코드를 볼 때 제목만으로도 기능을 얼추 유추할 수 있어야 합니다.

Reply:
네넵 다른 사람이 봐도 바로 감이 오도록 신경쓰도록 하겠습니다.